Makefile

```diff
@@ -63,3 +81,6 @@ format: $(GO_BIN)/gofumpt $(GO_BIN)/gci
 	@echo "==> Formatting project files"
 	gofumpt -w .
 	gci write -s standard -s default .
+
+mocks: $(MOCK_DIR)/*.go
```

Comment:
Should we add this as a dependency for `make test`? 🤔

Reply:
Yeah, that'd be a good idea